pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3385,6 +3385,69 @@ func Test_server_MultiInsert(t *testing.T) {
+				- case 1.3: MultiInsert 100 vector success (vector type is uint8)
+				- case 1.4: MultiInsert 100 vector success (vector type is float32)
+				- case 1.5: MultiInsert 0 vector fail (vector type is uint8)
+				- case 1.6: MultiInsert 0 vector fail (vector type is float32)
+				- case 2.1: MultiInsert 1 vector with different dimension fail (vector type is uint8)
+				- case 2.2: MultiInsert 1 vector with different dimension fail (vector type is float32)
+				- case 3.1: MultiInsert 100 vector with 1 vector with different dimension fail (vector type is uint8)
+				- case 3.2: MultiInsert 100 vector with 1 vector with different dimension fail (vector type is float32)
+				- case 3.3: MultiInsert 100 vector with 50 vector with different dimension fail (vector type is uint8)
+				- case 3.4: MultiInsert 100 vector with 50 vector with different dimension fail (vector type is float32)
+				- case 3.5: MultiInsert 100 vector with all vector with different dimension fail (vector type is uint8)
+				- case 3.6: MultiInsert 100 vector with all vector with different dimension fail (vector type is float32)
+
+		- Boundary Value Testing
+			- uint8, float32
+				- case 1.1: MultiInsert 100 vector with 0 value success (vector type is uint8)
+				- case 1.2: MultiInsert 100 vector with 0 value success (vector type is float32)
+				- case 2.1: MultiInsert 100 vector with min value success (vector type is uint8)
+				- case 2.2: MultiInsert 100 vector with min value success (vector type is float32)
+				- case 3.1: MultiInsert 100 vector with max value success (vector type is uint8)
+				- case 3.2: MultiInsert 100 vector with max value success (vector type is float32)
+				- case 4.1: MultiInsert 100 vector with 1 request with empty UUID fail (vector type is uint8)
+				- case 4.2: MultiInsert 100 vector with 1 request with empty UUID fail (vector type is float32)
+				- case 4.3: MultiInsert 100 vector with 50 request with empty UUID fail (vector type is uint8)
+				- case 4.4: MultiInsert 100 vector with 50 request with empty UUID fail (vector type is float32)
+				- case 4.5: MultiInsert 100 vector with all request with empty UUID fail (vector type is uint8)
+				- case 4.6: MultiInsert 100 vector with all request with empty UUID fail (vector type is float32)
+				- case 5.1: MultiInsert 100 vector with 1 vector with maximum dimension fail (vector type is uint8)
+				- case 5.2: MultiInsert 100 vector with 1 vector with maximum dimension fail (vector type is float32)
+				- case 5.3: MultiInsert 100 vector with 50 vector with maximum dimension fail (vector type is uint8)
+				- case 5.4: MultiInsert 100 vector with 50 vector with maximum dimension fail (vector type is float32)
+				- case 5.5: MultiInsert 100 vector with all vector with maximum dimension fail (vector type is uint8)
+				- case 5.6: MultiInsert 100 vector with all vector with maximum dimension fail (vector type is float32)
+
```

Comment:
I think Boundary Value Testing needs MultiInsert maximum vectors test.
However, the test may not run because of the RAM.